proof/refine/ARM/Tcb_R.thy

```diff
@@ -2791,29 +2791,205 @@ lemma decodeSetSchedParams_wf[wp]:
          simp)
   done *)
 
+(* FIXME RT: move to StateRelation *)
+lemma cap_rel_valid_fh:
+  "cap_relation a b \<Longrightarrow> valid_fault_handler a = isValidFaultHandler b"
+  apply (case_tac a
+         ; case_tac b
+         ; simp add: valid_fault_handler_def isValidFaultHandler_def)
+  apply (rule iffI
+         ; clarsimp simp: has_handler_rights_def split: bool.split_asm)
+  done
+
+(* FIXME RT: move to...? *)
```

Comment:
I guess Corres_UL? There's some similar lemmas in Corres_Method, but they involve the protected `corres_underlyingK`.